modules/astparser/src/QC_AstParser.qpp

```diff
@@ -54,7 +54,7 @@ AstParser::constructor() {
 *AstTree AstParser::parseFile(string filename) {
     ASTTree* tree = aph->parseFile(filename->getBuffer());
     if (!tree)
-        return nullptr;
+        return (AbstractQoreNode*) nullptr;
```

Comment:
please change this to `QoreValue()` - also in the other locations, thx - a global search and replace should do it it one go